sql/catalyst/src/main/java/org/apache/spark/sql/connector/expressions/GeneralScalarExpression.java

```diff
@@ -235,8 +235,8 @@ public String toString() {
     try {
       return builder.build(this);
     } catch (Throwable e) {
-      return name + "(" +
-        Arrays.stream(children).map(child -> child.toString()).reduce((a,b) -> a + "," + b) + ")";
```

Comment:
what's wrong with the previous code?

Reply:
The previous code let the toString display as `Option(...)`.